models/src/main/scala/coop/rchain/casper/PrettyPrinter.scala

```diff
@@ -38,12 +39,14 @@ object PrettyPrinter {
     }
 
   def buildString(d: ProcessedDeploy): String =
-    s"User: ${buildStringNoLimit(d.deploy.deployer)}, Cost: ${d.cost.toString} " +
+    s"User: ${buildStringNoLimit(d.deploy.pk.bytes)}, Cost: ${d.cost.toString} " +
       s"${buildString(d.deploy)}"
 
   def buildString(b: ByteString): String =
     limit(Base16.encode(b.toByteArray), 10)
 
+  def buildString(sd: Signed[DeployData]): String =
+    s"Sig: ${buildString(sd.sig)}, SigAlgorithm: ${sd.sigAlgorithm.name}"
   def buildString(d: DeployData): String =
```

Comment:
@zsluedem I think this is not called anymore and the whole `DeployData ...` is not printed. Is this what you see?

Reply:
You are right~! This is the right place to change.